Sources/Tracing/Span.swift

```diff
@@ -67,7 +70,7 @@ public protocol Span: AnyObject, _SwiftTracingSendableSpan {
     /// - Parameter time: The `DispatchWallTime` at which the span ended.
     ///
     /// - SeeAlso: `Span.end()` which automatically uses the "current" time.
-    func end(at time: DispatchWallTime)
+    func end(at time: Clock.Instant)
```

Comment:
And this way we got rid of the Dispatch types in API - which would have stayed with us "forever"